app/uk/gov/hmrc/helptosavestub/controllers/UserInfoAPIController.scala

```diff
@@ -79,11 +82,53 @@ class UserInfoAPIController extends BaseController {
   }
 
 
+  //DATA FOR AIR GAP TESTING
+  object AirGapTesting {
+    type Token = String
+
+    val scenario1User = UserInfo(Some("Sarah"), Some("Smith"), None, Some(Address("1 the street\n the place\n the town\n line 4\n line 5\n", Some("BN43 5QP"),
+      Some("United Kingdom"), Some("GB"))), Some(LocalDate.of(1999, 12, 12)), None, None, Some("sarah@smith.com"))
+
+    val scenerio2User = UserInfo(Some("Sarah"), Some("Smith"), None, Some(Address("1 the street\n the place", Some("BN43 5QP"),
+      Some("United Kingdom"), Some("GB"))), Some(LocalDate.of(1999, 12, 12)), None, None, Some("sarah@smith.com"))
+
+    val email = Random.alphanumeric.take(64).mkString("") + "@" + Random.alphanumeric.take(189).mkString("")
+
+    def randomString(length: Int): String = Random.alphanumeric.take(length).mkString("")
+
+    val scenerio3User = UserInfo(Some("Sarahjacquelinefredricktom"), Some("SmiththissurnameisthreehundredcharacterslongsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithSmiththissurnameisthreehundredcharacterslongsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithsmithSmiththissurnameisthreehundredcharacterslongsmithsmithsmithsmithsmithsmi"),
```

Comment:
You could create a `Stream` which returns just letters like this:
```scala 
def randomAlphaNumericString(length: Int): String = Random.alphanumeric.take(length).mkString("")
def randomAlphaString(length: Int): String = Random.alphanumeric.filter(_.isLetter).take(length).mkString("")
```
and use it to create the long name here